src/app/[locale]/components/edit/[id]/components/Releases.tsx

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) TOSHIBA CORPORATION, 2023. Part of the SW360 Frontend Project.
+// Copyright (C) Toshiba Software Development (Vietnam) Co., Ltd., 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+'use client'
+
+import { useEffect, useState, useCallback } from 'react'
+import CommonUtils from '@/utils/common.utils'
+import Link from 'next/link'
+import { useTranslations } from 'next-intl'
+import { COMMON_NAMESPACE } from '@/object-types/Constants'
+import ApiUtils from '@/utils/api/api.util'
+import HttpStatus from '@/object-types/enums/HttpStatus'
+import { signOut } from 'next-auth/react'
+import { notFound } from 'next/navigation'
+import { Session } from '@/object-types/Session'
+import ReleaseLink from '@/object-types/ReleaseLink'
+
+import { Table, _ } from '@/components/sw360'
+
+interface Props {
+    session: Session
+    componentId: string
+}
+
+const Releases = ({ session, componentId }: Props) => {
+    const t = useTranslations(COMMON_NAMESPACE)
+    const [data, setData] = useState([])
+
+    const fetchData: any = useCallback(
+        async (url: string) => {
+            const response = await ApiUtils.GET(url, session.user.access_token)
+            if (response.status == HttpStatus.OK) {
+                const data = await response.json()
+                return data
+            } else if (response.status == HttpStatus.UNAUTHORIZED) {
+                signOut()
+            } else {
+                notFound()
+            }
+        },
+        [session.user.access_token]
+    )
+
+    useEffect(() => {
+        fetchData(`components/${componentId}/releases`).then((releaseLinks: any) => {
+            if (
+                !CommonUtils.isNullOrUndefined(releaseLinks['_embedded']) &&
+                !CommonUtils.isNullOrUndefined(releaseLinks['_embedded']['sw360:releaseLinks'])
+            ) {
+                const data = releaseLinks['_embedded']['sw360:releaseLinks'].map((item: ReleaseLink) => [
+                    item.name,
+                    [item.id, item.version],
+                ])
+                setData(data)
+            }
+        })
+    }, [])
+
```

Comment:
@hoangnt2 @amritkv Can we get in a common way to fetch data ? Right now i'm using @hoangnt2 APIUtils on my license code, but there are at least three different ways of fetching data.
We should set for one otherwise will become way too hard to debug